frontend/talentsearch/src/js/api/workLocationPreferenceOperations.graphql

```diff
@@ -0,0 +1,15 @@
+query WorkLocationPreference($id: ID!) {
+  user(id: $id) {
+    id
+    email
```

Comment:
Ya. Good question. I added email to resolve an error when I used user instead of me. I was thinking last night to remove the email  then forgot. Hopefuly it will work. let you know soon.

Reply:
email removed